funcbench/bench.go

```diff
@@ -108,77 +108,30 @@ func (b *Benchmarker) exec(ctx context.Context, pkgRoot string, commit plumbing.
 	return fn, nil
 }
 
-func parseFile(path string) (parse.Set, error) {
-	f, err := os.Open(path)
-	if err != nil {
-		return nil, err
-	}
-	defer f.Close()
-	bb, err := parse.ParseSet(f)
-	if err != nil {
-		return nil, err
-	}
-	return bb, nil
-}
-
-func (b *Benchmarker) compareBenchmarks(beforeFile, afterFile string) ([]BenchCmp, error) {
-	before, err := parseFile(beforeFile)
-	if err != nil {
-		return nil, errors.Wrapf(err, "open %s", beforeFile)
-	}
-
-	after, err := parseFile(afterFile)
-	if err != nil {
-		return nil, errors.Wrapf(err, "open %s", afterFile)
-	}
-
-	// Use benchcmp library directly.
-	// TODO(bwplotka): benchstat is new thing - we might add choice for funcbench to choose from? (:
```

Comment:
Comparison: 

> note: this benchmark results are almost the same as there were not a lot of changes related to the benchmark being added.

Funcbench runs using this PR(`benchstat`):
- run1: https://github.com/prometheus-community/prometheus/pull/71#issuecomment-656452591
- run2: https://github.com/prometheus-community/prometheus/pull/71#issuecomment-656469104

Funcbench runs using master(`benchcmp`):
- run1: https://github.com/prometheus-community/prometheus/pull/71#issuecomment-656476903
- run2: https://github.com/prometheus-community/prometheus/pull/71#issuecomment-656482497

I think this PR adds some really nice stuff, but concerned about @bwplotka's comment on supporting both of them. I was wondering if we still need to keep compatibility with `benchcmp`? 

cc: @bwplotka

Reply:
benchstat is better but I use `-delta-none` (: